tools/create-yorkie-app/index.ts

```diff
@@ -262,15 +261,15 @@ function copyDir(srcDir: string, destDir: string) {
 
 function isEmpty(path: string) {
   const files = fs.readdirSync(path);
-  return files.length === 0 || (files.length === 1 && files[0] === '.git');
+  return !files.length || !files.filter((file) => /^[^.]+/.test(file)).length;
 }
 
 function emptyDir(dir: string) {
   if (!fs.existsSync(dir)) {
     return;
   }
   for (const file of fs.readdirSync(dir)) {
-    if (file === '.git') {
+    if (/^\./.test(file)) {
```

Comment:
Also, avoid declaring regular expressions inline within functions or loops. In this case, a new regex instance is created for every function call (filter callback) or loop.

If the expressions are the same, create only one instance of the regular expression to use.

See https://tc39.es/ecma262/#sec-literals-regular-expression-literals